src/Symfony/Component/Workflow/Transition.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class Transition
+{
+    private $name;
+    private $froms;
+    private $tos;
+
+    public function __construct($name, $froms, $tos)
+    {
+        $this->name = $name;
+        $this->froms = (array) $froms;
```

Comment:
Because it allows you to run `new Transition('the name', 'from', 'to')` instead of `new Transition('the name', ['from'], ['to'])`